### ISSUE TITLE
Missing final new line in amptk_mock1.fa and amptk_synmock.fa

### ISSUE THREAD

**Issue by peterjc** [nextgenusfs/amptk#88]
Verified with the OSF archive and a fresh git clone as of commit 0873598ec519a3db9e17fc32bdaddbd87de21c24

```
$ for F in amptk_*.fa; do echo $F; hexdump -C $F | tail; done
amptk_mock1.fa
00001810  43 47 47 43 43 54 43 47  54 43 47 43 47 47 43 43  |CGGCCTCGTCGCGGCC|
00001820  54 43 43 54 43 54 47 43  47 54 41 47 54 41 47 43  |TCCTCTGCGTAGTAGC|
00001830  41 43 41 41 43 43 54 43  47 43 41 41 47 43 47 47  |ACAACCTCGCAAGCGG|
00001840  47 41 47 43 47 43 47 41  43 47 43 47 47 43 43 41  |GAGCGCGACGCGGCCA|
00001850  43 54 47 43 43 47 54 41  41 41 41 43 47 43 43 43  |CTGCCGTAAAACGCCC|
00001860  41 41 43 54 54 54 54 54  54 41 47 41 47 54 54 47  |AACTTTTTTAGAGTTG|
00001870  41 43 43 54 43 47 47 41  54 43 41 47 47 54 41 47  |ACCTCGGATCAGGTAG|
00001880  47 41 41 54 41 43 43 43  47 43 54 47 41 41 43 54  |GAATACCCGCTGAACT|
00001890  54 41 41                                          |TAA|
00001893
amptk_mock2.fa
00003e60  41 43 54 54 54 41 43 41  43 43 54 54 47 43 43 54  |ACTTTACACCTTGCCT|
00003e70  54 43 41 47 43 47 0a 54  47 41 54 41 41 54 54 47  |TCAGCG.TGATAATTG|
00003e80  54 43 54 41 43 47 54 54  47 54 43 47 43 41 43 54  |TCTACGTTGTCGCACT|
00003e90  41 54 47 54 41 54 54 43  41 54 41 43 47 54 54 43  |ATGTATTCATACGTTC|
00003ea0  54 43 47 43 54 54 41 43  41 0a 41 54 43 47 54 43  |TCGCTTACA.ATCGTC|
00003eb0  54 43 54 43 54 47 41 47  41 43 41 54 54 54 54 54  |TCTCTGAGACATTTTT|
00003ec0  54 43 54 41 41 43 54 54  43 54 47 41 54 43 54 43  |TCTAACTTCTGATCTC|
00003ed0  41 41 41 54 43 41 47 47  54 41 47 47 0a 41 43 54  |AAATCAGGTAGG.ACT|
00003ee0  41 43 43 43 47 43 54 47  41 41 43 54 54 41 41 0a  |ACCCGCTGAACTTAA.|
00003ef0
amptk_mock3.fa
000036e0  41 43 41 43 43 54 54 47  43 43 54 54 43 41 47 43  |ACACCTTGCCTTCAGC|
000036f0  47 0a 54 47 41 54 41 41  54 54 47 54 43 54 41 43  |G.TGATAATTGTCTAC|
00003700  47 54 54 47 54 43 47 43  41 43 54 41 54 47 54 41  |GTTGTCGCACTATGTA|
00003710  54 54 43 41 54 41 43 47  54 54 43 54 43 47 43 54  |TTCATACGTTCTCGCT|
00003720  54 41 43 41 0a 41 54 43  47 54 43 54 43 54 43 54  |TACA.ATCGTCTCTCT|
00003730  47 41 47 41 43 41 54 54  54 54 54 54 43 54 41 41  |GAGACATTTTTTCTAA|
00003740  43 54 54 43 54 47 41 54  43 54 43 41 41 41 54 43  |CTTCTGATCTCAAATC|
00003750  41 47 47 54 41 47 47 0a  41 43 54 41 43 43 43 47  |AGGTAGG.ACTACCCG|
00003760  43 54 47 41 41 43 54 54  41 41 0a                 |CTGAACTTAA.|
0000376b
amptk_synmock.fa
00001e60  54 43 54 41 47 47 54 41  54 43 41 41 54 43 47 54  |TCTAGGTATCAATCGT|
00001e70  54 54 43 41 43 41 54 41  54 41 43 41 54 47 43 41  |TTCACATATACATGCA|
00001e80  41 41 54 43 54 0a 47 54  41 41 54 47 43 43 54 41  |AATCT.GTAATGCCTA|
00001e90  54 47 43 43 47 41 54 47  43 43 43 54 43 41 47 43  |TGCCGATGCCCTCAGC|
00001ea0  43 47 41 47 54 47 41 54  54 47 47 41 47 41 54 54  |CGAGTGATTGGAGATT|
00001eb0  41 47 47 47 54 47 41 43  43 54 43 41 41 41 54 43  |AGGGTGACCTCAAATC|
00001ec0  41 47 47 54 41 47 47 41  47 54 41 43 43 43 47 43  |AGGTAGGAGTACCCGC|
00001ed0  54 47 41 41 43 54 54 41  41 47 43 41 54 41 54 43  |TGAACTTAAGCATATC|
00001ee0  41 41 54 41 41 47 43 47  47 41 0a 47 47 41        |AATAAGCGGA.GGA|
00001eee
```

Note the missing 0x0a new line character in two of the four FASTA files.

This can cause subtle bugs if processed with command line tools, especially anything using like ``cat amptk_*.fa``

**Comment by hyphaltip** [comment]
a pull request on this would be welcomed I am sure peter!

**Comment by peterjc** [comment]
Thanks Jason 👍 

I'd have been have to fix that and the typo in #86 but you beat me to it. Some of my other queries are less simple...
